lib/matplotlib/testing/decorators.py

```diff
@@ -462,3 +465,24 @@ def _image_directories(func):
     result_dir = Path().resolve() / "result_images" / module_path.stem
     result_dir.mkdir(parents=True, exist_ok=True)
     return baseline_dir, result_dir
+
+
+def skipif_font_missing(fonts):
+    """
+    Decorator to skip a test if any of the specified fonts are missing.
+
+    Usage example:
+        @skipif_font_missing(['Dejavu Sans', 'Noto SansTC'])
+        def test_font(family_name):
+            # testing code
+    """
+    def decorator(func):
+        def wrapper(family_name):
```

Comment:
Should probably use `functools.wraps` here, as we do not wish to assume the signature of the wrapped function